rust/automerge-wasm/src/lib.rs

```diff
@@ -804,6 +804,43 @@ impl Automerge {
         Ok(self.export_object(&obj, obj_type.into(), heads.as_ref(), &meta)?)
     }
 
+    #[wasm_bindgen(js_name = getCursor)]
+    pub fn get_cursor(
+        &mut self,
+        obj: JsValue,
+        index: f64,
+        heads: Option<Array>,
+    ) -> Result<String, error::Cursor> {
+        let (obj, obj_type) = self.import(obj).unwrap_or((ROOT, am::ObjType::Map));
+        if obj_type != am::ObjType::Text {
+            return Err(error::Cursor::InvalidObjType(obj_type));
+        }
+        let index = index as usize;
+        let heads = get_heads(heads)?;
+        let cursor = self.doc.get_cursor(obj, index, heads.as_deref())?;
+        Ok(cursor.to_string())
```

Comment:
Rather than using the `Display` implementation here should we consider using the base64 encoding of the serialization format we defined in `automerge/src/cursor.rs`? The advantage of that approach is that we have a version tag in the format so if we want to change how cursors are represented (for ex. to include a branch ID) we can do so transparently.

Reply:
At the end of the day it's on opid.  Being able to see a string on the js side and having it look like "10@aabbcc", seeing the actor and what not, feels consistent clear and transparent and consistent with how we're handling object ids.